e2e/tests/api_tests/sdo/sdo_test.js

```diff
@@ -233,6 +233,29 @@ Scenario.skip('1v1 full defence unspecified - legal advisor declares SDO unsuita
   }
 });
 
+Feature('CCD 1v1 API test @api-finalOrders');
```

Comment:
Is this in the right place? You're following the dafault Judgement route, but this is in the SDO scenarios?
Also, it is under the _nightly_ test section that starts at line 100, is that intended?